From e0e6c1b2f1cbc5bad3202231feb634cac06becd2 Mon Sep 17 00:00:00 2001
From: Dan Nica <dan.nica@nxp.com>
Date: Fri, 5 Jun 2020 14:55:49 +0300
Subject: [PATCH 147/269] s32g: Change the directory structure and filename
 prefixes

Replace 's32g274a_' filename prefixes with 's32g_' and move 'plat/s32g'
to 'plat/nxp/s32g'

Issue: ALB-4935
Upstream-Status: Pending 

Signed-off-by: Dan Nica <dan.nica@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../s32g/i2c/{s32g274a_i2c.c => s32g_i2c.c}   |  2 +-
 drivers/nxp/s32g/io/io_memmap.c               |  2 +-
 .../s32g/mmc/{s32g274a_mmc.c => s32g_mmc.c}   |  0
 .../s32g/i2c/{s32g274a_i2c.h => s32g_i2c.h}   |  0
 .../s32g/mmc/{s32g274a_mmc.h => s32g_mmc.h}   |  0
 include/drivers/nxp/s32g/pmic/vr5510.h        |  2 +-
 plat/{ => nxp}/s32g/bl31_lowlevel.S           |  0
 plat/{ => nxp}/s32g/bl31_sram/.gitignore      |  0
 plat/{ => nxp}/s32g/bl31_sram/bl31SRAM.ld.S   |  0
 plat/{ => nxp}/s32g/bl31_sram/bl31_sram.mk    | 14 ++---
 .../s32g/bl31_sram/bl31sram_entrypoint.S      |  0
 plat/{ => nxp}/s32g/bl31_sram/bl31sram_main.c |  0
 .../s32g/bl31_sram/include/bl31_sram.h        |  0
 plat/{ => nxp}/s32g/bl31_ssram/.gitignore     |  0
 plat/{ => nxp}/s32g/bl31_ssram/bl31SSRAM.ld.S |  0
 plat/{ => nxp}/s32g/bl31_ssram/bl31_ssram.mk  | 16 +++---
 .../s32g/bl31_ssram/bl31ssram_entrypoint.S    |  0
 .../s32g/bl31_ssram/bl31ssram_main.c          |  0
 .../s32g/bl31_ssram/bl31ssram_stacks.S        |  0
 .../s32g/bl31_ssram/include/bl31_ssram.h      |  0
 .../s32g/bl31_ssram/include/ssram_mailbox.h   |  0
 plat/{ => nxp}/s32g/include/plat_macros.S     |  0
 plat/{ => nxp}/s32g/include/platform_def.h    |  0
 plat/{ => nxp}/s32g/include/s32g_clocks.h     |  0
 plat/{ => nxp}/s32g/include/s32g_dt.h         |  0
 .../s32g/include/s32g_edma.h}                 |  0
 plat/{ => nxp}/s32g/include/s32g_ivt.h        |  0
 .../{ => nxp}/s32g/include/s32g_linflexuart.h |  0
 plat/{ => nxp}/s32g/include/s32g_lowlevel.h   |  0
 plat/{ => nxp}/s32g/include/s32g_mc_me.h      |  0
 plat/{ => nxp}/s32g/include/s32g_mc_rgm.h     |  0
 plat/{ => nxp}/s32g/include/s32g_ncore.h      |  0
 plat/{ => nxp}/s32g/include/s32g_pinctrl.h    |  0
 .../s32g/include/s32g_pm.h}                   |  0
 plat/{ => nxp}/s32g/include/s32g_storage.h    |  0
 plat/{ => nxp}/s32g/include/s32g_xrdc.h       |  0
 plat/{ => nxp}/s32g/platform.mk               | 52 +++++++++----------
 .../s32g/s32g_bl2_el3.c}                      |  0
 .../s32g274a_bl31.c => nxp/s32g/s32g_bl31.c}  |  4 +-
 .../s32g/s32g_bl_common.c}                    |  0
 plat/{ => nxp}/s32g/s32g_clocks.c             |  0
 plat/{ => nxp}/s32g/s32g_common.mk            |  0
 plat/{ => nxp}/s32g/s32g_dt.c                 |  0
 .../s32g274a_edma.c => nxp/s32g/s32g_edma.c}  |  2 +-
 plat/{ => nxp}/s32g/s32g_linflexuart.S        |  0
 plat/{ => nxp}/s32g/s32g_lowlevel_bl2.S       |  0
 plat/{ => nxp}/s32g/s32g_lowlevel_common.S    |  2 +-
 plat/{ => nxp}/s32g/s32g_mc_me.c              |  0
 plat/{ => nxp}/s32g/s32g_ncore.c              |  0
 plat/{ => nxp}/s32g/s32g_pinctrl.c            |  0
 .../s32g274a_pm.c => nxp/s32g/s32g_pm.c}      |  2 +-
 plat/{ => nxp}/s32g/s32g_psci.c               |  0
 .../s32g/s32g_storage.c}                      |  2 +-
 plat/{ => nxp}/s32g/s32g_xrdc.c               |  0
 54 files changed, 50 insertions(+), 50 deletions(-)
 rename drivers/nxp/s32g/i2c/{s32g274a_i2c.c => s32g_i2c.c} (99%)
 rename drivers/nxp/s32g/mmc/{s32g274a_mmc.c => s32g_mmc.c} (100%)
 rename include/drivers/nxp/s32g/i2c/{s32g274a_i2c.h => s32g_i2c.h} (100%)
 rename include/drivers/nxp/s32g/mmc/{s32g274a_mmc.h => s32g_mmc.h} (100%)
 rename plat/{ => nxp}/s32g/bl31_lowlevel.S (100%)
 rename plat/{ => nxp}/s32g/bl31_sram/.gitignore (100%)
 rename plat/{ => nxp}/s32g/bl31_sram/bl31SRAM.ld.S (100%)
 rename plat/{ => nxp}/s32g/bl31_sram/bl31_sram.mk (59%)
 rename plat/{ => nxp}/s32g/bl31_sram/bl31sram_entrypoint.S (100%)
 rename plat/{ => nxp}/s32g/bl31_sram/bl31sram_main.c (100%)
 rename plat/{ => nxp}/s32g/bl31_sram/include/bl31_sram.h (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/.gitignore (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/bl31SSRAM.ld.S (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/bl31_ssram.mk (54%)
 rename plat/{ => nxp}/s32g/bl31_ssram/bl31ssram_entrypoint.S (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/bl31ssram_main.c (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/bl31ssram_stacks.S (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/include/bl31_ssram.h (100%)
 rename plat/{ => nxp}/s32g/bl31_ssram/include/ssram_mailbox.h (100%)
 rename plat/{ => nxp}/s32g/include/plat_macros.S (100%)
 rename plat/{ => nxp}/s32g/include/platform_def.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_clocks.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_dt.h (100%)
 rename plat/{s32g/include/s32g274a_edma.h => nxp/s32g/include/s32g_edma.h} (100%)
 rename plat/{ => nxp}/s32g/include/s32g_ivt.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_linflexuart.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_lowlevel.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_mc_me.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_mc_rgm.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_ncore.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_pinctrl.h (100%)
 rename plat/{s32g/include/s32g274a_pm.h => nxp/s32g/include/s32g_pm.h} (100%)
 rename plat/{ => nxp}/s32g/include/s32g_storage.h (100%)
 rename plat/{ => nxp}/s32g/include/s32g_xrdc.h (100%)
 rename plat/{ => nxp}/s32g/platform.mk (79%)
 rename plat/{s32g/s32g274a_bl2_el3.c => nxp/s32g/s32g_bl2_el3.c} (100%)
 rename plat/{s32g/s32g274a_bl31.c => nxp/s32g/s32g_bl31.c} (99%)
 rename plat/{s32g/s32g274a_bl_common.c => nxp/s32g/s32g_bl_common.c} (100%)
 rename plat/{ => nxp}/s32g/s32g_clocks.c (100%)
 rename plat/{ => nxp}/s32g/s32g_common.mk (100%)
 rename plat/{ => nxp}/s32g/s32g_dt.c (100%)
 rename plat/{s32g/s32g274a_edma.c => nxp/s32g/s32g_edma.c} (98%)
 rename plat/{ => nxp}/s32g/s32g_linflexuart.S (100%)
 rename plat/{ => nxp}/s32g/s32g_lowlevel_bl2.S (100%)
 rename plat/{ => nxp}/s32g/s32g_lowlevel_common.S (99%)
 rename plat/{ => nxp}/s32g/s32g_mc_me.c (100%)
 rename plat/{ => nxp}/s32g/s32g_ncore.c (100%)
 rename plat/{ => nxp}/s32g/s32g_pinctrl.c (100%)
 rename plat/{s32g/s32g274a_pm.c => nxp/s32g/s32g_pm.c} (99%)
 rename plat/{ => nxp}/s32g/s32g_psci.c (100%)
 rename plat/{s32g/s32g274a_storage.c => nxp/s32g/s32g_storage.c} (98%)
 rename plat/{ => nxp}/s32g/s32g_xrdc.c (100%)

diff --git a/drivers/nxp/s32g/i2c/s32g274a_i2c.c b/drivers/nxp/s32g/i2c/s32g_i2c.c
similarity index 99%
rename from drivers/nxp/s32g/i2c/s32g274a_i2c.c
rename to drivers/nxp/s32g/i2c/s32g_i2c.c
index 9057d70ca..e5b146b52 100644
--- a/drivers/nxp/s32g/i2c/s32g274a_i2c.c
+++ b/drivers/nxp/s32g/i2c/s32g_i2c.c
@@ -18,7 +18,7 @@
 #include <lib/mmio.h>
 #include <libfdt.h>
 #include <drivers/delay_timer.h>
-#include <drivers/nxp/s32g/i2c/s32g274a_i2c.h>
+#include <drivers/nxp/s32g/i2c/s32g_i2c.h>
 #include "s32g_clocks.h"
 #include "s32g_dt.h"
 
diff --git a/drivers/nxp/s32g/io/io_memmap.c b/drivers/nxp/s32g/io/io_memmap.c
index ab4fc937d..0e016b407 100644
--- a/drivers/nxp/s32g/io/io_memmap.c
+++ b/drivers/nxp/s32g/io/io_memmap.c
@@ -12,7 +12,7 @@
 #include <drivers/io/io_memmap.h>
 #include <drivers/io/io_storage.h>
 #include <lib/utils.h>
-#include <s32g274a_edma.h>
+#include <s32g_edma.h>
 
 /* As we need to be able to keep state for seek, only one file can be open
  * at a time. Make this a structure and point to the entity->info. When we
diff --git a/drivers/nxp/s32g/mmc/s32g274a_mmc.c b/drivers/nxp/s32g/mmc/s32g_mmc.c
similarity index 100%
rename from drivers/nxp/s32g/mmc/s32g274a_mmc.c
rename to drivers/nxp/s32g/mmc/s32g_mmc.c
diff --git a/include/drivers/nxp/s32g/i2c/s32g274a_i2c.h b/include/drivers/nxp/s32g/i2c/s32g_i2c.h
similarity index 100%
rename from include/drivers/nxp/s32g/i2c/s32g274a_i2c.h
rename to include/drivers/nxp/s32g/i2c/s32g_i2c.h
diff --git a/include/drivers/nxp/s32g/mmc/s32g274a_mmc.h b/include/drivers/nxp/s32g/mmc/s32g_mmc.h
similarity index 100%
rename from include/drivers/nxp/s32g/mmc/s32g274a_mmc.h
rename to include/drivers/nxp/s32g/mmc/s32g_mmc.h
diff --git a/include/drivers/nxp/s32g/pmic/vr5510.h b/include/drivers/nxp/s32g/pmic/vr5510.h
index ad24e50e5..59b14f1a9 100644
--- a/include/drivers/nxp/s32g/pmic/vr5510.h
+++ b/include/drivers/nxp/s32g/pmic/vr5510.h
@@ -6,7 +6,7 @@
 #define VR5510_PMIC_H
 
 #include <lib/utils_def.h>
-#include "i2c/s32g274a_i2c.h"
+#include "i2c/s32g_i2c.h"
 
 #define VR5510_MU_NAME		"vr5510"
 #define VR5510_FSU_NAME		"vr5510_fsu"
diff --git a/plat/s32g/bl31_lowlevel.S b/plat/nxp/s32g/bl31_lowlevel.S
similarity index 100%
rename from plat/s32g/bl31_lowlevel.S
rename to plat/nxp/s32g/bl31_lowlevel.S
diff --git a/plat/s32g/bl31_sram/.gitignore b/plat/nxp/s32g/bl31_sram/.gitignore
similarity index 100%
rename from plat/s32g/bl31_sram/.gitignore
rename to plat/nxp/s32g/bl31_sram/.gitignore
diff --git a/plat/s32g/bl31_sram/bl31SRAM.ld.S b/plat/nxp/s32g/bl31_sram/bl31SRAM.ld.S
similarity index 100%
rename from plat/s32g/bl31_sram/bl31SRAM.ld.S
rename to plat/nxp/s32g/bl31_sram/bl31SRAM.ld.S
diff --git a/plat/s32g/bl31_sram/bl31_sram.mk b/plat/nxp/s32g/bl31_sram/bl31_sram.mk
similarity index 59%
rename from plat/s32g/bl31_sram/bl31_sram.mk
rename to plat/nxp/s32g/bl31_sram/bl31_sram.mk
index 07d4ea854..6a688b441 100644
--- a/plat/s32g/bl31_sram/bl31_sram.mk
+++ b/plat/nxp/s32g/bl31_sram/bl31_sram.mk
@@ -5,18 +5,18 @@
 #
 
 BL31SRAM_SOURCES = plat/common/aarch64/platform_up_stack.S \
-		   plat/s32g/bl31_sram/bl31sram_entrypoint.S \
-		   plat/s32g/bl31_sram/bl31sram_main.c \
-		   plat/s32g/s32g_clocks.c \
-		   plat/s32g/s32g_linflexuart.S \
-		   plat/s32g/s32g_mc_me.c \
+		   plat/nxp/s32g/bl31_sram/bl31sram_entrypoint.S \
+		   plat/nxp/s32g/bl31_sram/bl31sram_main.c \
+		   plat/nxp/s32g/s32g_clocks.c \
+		   plat/nxp/s32g/s32g_linflexuart.S \
+		   plat/nxp/s32g/s32g_mc_me.c \
 		   ${DDR_DRV}/ddrss.c \
 
 BL31SRAM_ARRAY_NAME ?= bl31sram
 BL31SRAM_ARRAY_LEN  ?= bl31sram_len
 
-BL31SRAM_SRC_DUMP   := plat/s32g/bl31_sram/bl31_sram.c
-BL31SRAM_LINKERFILE := plat/s32g/bl31_sram/bl31SRAM.ld.S
+BL31SRAM_SRC_DUMP   := plat/nxp/s32g/bl31_sram/bl31_sram.c
+BL31SRAM_LINKERFILE := plat/nxp/s32g/bl31_sram/bl31SRAM.ld.S
 
 $(eval $(call MAKE_BL,31SRAM))
 
diff --git a/plat/s32g/bl31_sram/bl31sram_entrypoint.S b/plat/nxp/s32g/bl31_sram/bl31sram_entrypoint.S
similarity index 100%
rename from plat/s32g/bl31_sram/bl31sram_entrypoint.S
rename to plat/nxp/s32g/bl31_sram/bl31sram_entrypoint.S
diff --git a/plat/s32g/bl31_sram/bl31sram_main.c b/plat/nxp/s32g/bl31_sram/bl31sram_main.c
similarity index 100%
rename from plat/s32g/bl31_sram/bl31sram_main.c
rename to plat/nxp/s32g/bl31_sram/bl31sram_main.c
diff --git a/plat/s32g/bl31_sram/include/bl31_sram.h b/plat/nxp/s32g/bl31_sram/include/bl31_sram.h
similarity index 100%
rename from plat/s32g/bl31_sram/include/bl31_sram.h
rename to plat/nxp/s32g/bl31_sram/include/bl31_sram.h
diff --git a/plat/s32g/bl31_ssram/.gitignore b/plat/nxp/s32g/bl31_ssram/.gitignore
similarity index 100%
rename from plat/s32g/bl31_ssram/.gitignore
rename to plat/nxp/s32g/bl31_ssram/.gitignore
diff --git a/plat/s32g/bl31_ssram/bl31SSRAM.ld.S b/plat/nxp/s32g/bl31_ssram/bl31SSRAM.ld.S
similarity index 100%
rename from plat/s32g/bl31_ssram/bl31SSRAM.ld.S
rename to plat/nxp/s32g/bl31_ssram/bl31SSRAM.ld.S
diff --git a/plat/s32g/bl31_ssram/bl31_ssram.mk b/plat/nxp/s32g/bl31_ssram/bl31_ssram.mk
similarity index 54%
rename from plat/s32g/bl31_ssram/bl31_ssram.mk
rename to plat/nxp/s32g/bl31_ssram/bl31_ssram.mk
index debe46387..a5dde47da 100644
--- a/plat/s32g/bl31_ssram/bl31_ssram.mk
+++ b/plat/nxp/s32g/bl31_ssram/bl31_ssram.mk
@@ -4,12 +4,12 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-BL31SSRAM_SOURCES =  plat/s32g/bl31_ssram/bl31ssram_stacks.S \
-		     plat/s32g/bl31_ssram/bl31ssram_entrypoint.S \
-		     plat/s32g/bl31_ssram/bl31ssram_main.c \
-		     plat/s32g/s32g_linflexuart.S \
-		     plat/s32g/s32g_clocks.c \
-		     plat/s32g/s32g_mc_me.c \
+BL31SSRAM_SOURCES =  plat/nxp/s32g/bl31_ssram/bl31ssram_stacks.S \
+		     plat/nxp/s32g/bl31_ssram/bl31ssram_entrypoint.S \
+		     plat/nxp/s32g/bl31_ssram/bl31ssram_main.c \
+		     plat/nxp/s32g/s32g_linflexuart.S \
+		     plat/nxp/s32g/s32g_clocks.c \
+		     plat/nxp/s32g/s32g_mc_me.c \
 		     ${DDR_DRV}/ddrss.c \
 		     ${DDR_DRV}/ddrss_regconf.c \
 		     ${LIBC_SRCS}
@@ -17,8 +17,8 @@ BL31SSRAM_SOURCES =  plat/s32g/bl31_ssram/bl31ssram_stacks.S \
 BL31SSRAM_ARRAY_NAME ?= bl31ssram
 BL31SSRAM_ARRAY_LEN  ?= bl31ssram_len
 
-BL31SSRAM_SRC_DUMP   := plat/s32g/bl31_ssram/bl31_ssram.c
-BL31SSRAM_LINKERFILE := plat/s32g/bl31_ssram/bl31SSRAM.ld.S
+BL31SSRAM_SRC_DUMP   := plat/nxp/s32g/bl31_ssram/bl31_ssram.c
+BL31SSRAM_LINKERFILE := plat/nxp/s32g/bl31_ssram/bl31SSRAM.ld.S
 
 $(eval $(call MAKE_BL,31SSRAM))
 
diff --git a/plat/s32g/bl31_ssram/bl31ssram_entrypoint.S b/plat/nxp/s32g/bl31_ssram/bl31ssram_entrypoint.S
similarity index 100%
rename from plat/s32g/bl31_ssram/bl31ssram_entrypoint.S
rename to plat/nxp/s32g/bl31_ssram/bl31ssram_entrypoint.S
diff --git a/plat/s32g/bl31_ssram/bl31ssram_main.c b/plat/nxp/s32g/bl31_ssram/bl31ssram_main.c
similarity index 100%
rename from plat/s32g/bl31_ssram/bl31ssram_main.c
rename to plat/nxp/s32g/bl31_ssram/bl31ssram_main.c
diff --git a/plat/s32g/bl31_ssram/bl31ssram_stacks.S b/plat/nxp/s32g/bl31_ssram/bl31ssram_stacks.S
similarity index 100%
rename from plat/s32g/bl31_ssram/bl31ssram_stacks.S
rename to plat/nxp/s32g/bl31_ssram/bl31ssram_stacks.S
diff --git a/plat/s32g/bl31_ssram/include/bl31_ssram.h b/plat/nxp/s32g/bl31_ssram/include/bl31_ssram.h
similarity index 100%
rename from plat/s32g/bl31_ssram/include/bl31_ssram.h
rename to plat/nxp/s32g/bl31_ssram/include/bl31_ssram.h
diff --git a/plat/s32g/bl31_ssram/include/ssram_mailbox.h b/plat/nxp/s32g/bl31_ssram/include/ssram_mailbox.h
similarity index 100%
rename from plat/s32g/bl31_ssram/include/ssram_mailbox.h
rename to plat/nxp/s32g/bl31_ssram/include/ssram_mailbox.h
diff --git a/plat/s32g/include/plat_macros.S b/plat/nxp/s32g/include/plat_macros.S
similarity index 100%
rename from plat/s32g/include/plat_macros.S
rename to plat/nxp/s32g/include/plat_macros.S
diff --git a/plat/s32g/include/platform_def.h b/plat/nxp/s32g/include/platform_def.h
similarity index 100%
rename from plat/s32g/include/platform_def.h
rename to plat/nxp/s32g/include/platform_def.h
diff --git a/plat/s32g/include/s32g_clocks.h b/plat/nxp/s32g/include/s32g_clocks.h
similarity index 100%
rename from plat/s32g/include/s32g_clocks.h
rename to plat/nxp/s32g/include/s32g_clocks.h
diff --git a/plat/s32g/include/s32g_dt.h b/plat/nxp/s32g/include/s32g_dt.h
similarity index 100%
rename from plat/s32g/include/s32g_dt.h
rename to plat/nxp/s32g/include/s32g_dt.h
diff --git a/plat/s32g/include/s32g274a_edma.h b/plat/nxp/s32g/include/s32g_edma.h
similarity index 100%
rename from plat/s32g/include/s32g274a_edma.h
rename to plat/nxp/s32g/include/s32g_edma.h
diff --git a/plat/s32g/include/s32g_ivt.h b/plat/nxp/s32g/include/s32g_ivt.h
similarity index 100%
rename from plat/s32g/include/s32g_ivt.h
rename to plat/nxp/s32g/include/s32g_ivt.h
diff --git a/plat/s32g/include/s32g_linflexuart.h b/plat/nxp/s32g/include/s32g_linflexuart.h
similarity index 100%
rename from plat/s32g/include/s32g_linflexuart.h
rename to plat/nxp/s32g/include/s32g_linflexuart.h
diff --git a/plat/s32g/include/s32g_lowlevel.h b/plat/nxp/s32g/include/s32g_lowlevel.h
similarity index 100%
rename from plat/s32g/include/s32g_lowlevel.h
rename to plat/nxp/s32g/include/s32g_lowlevel.h
diff --git a/plat/s32g/include/s32g_mc_me.h b/plat/nxp/s32g/include/s32g_mc_me.h
similarity index 100%
rename from plat/s32g/include/s32g_mc_me.h
rename to plat/nxp/s32g/include/s32g_mc_me.h
diff --git a/plat/s32g/include/s32g_mc_rgm.h b/plat/nxp/s32g/include/s32g_mc_rgm.h
similarity index 100%
rename from plat/s32g/include/s32g_mc_rgm.h
rename to plat/nxp/s32g/include/s32g_mc_rgm.h
diff --git a/plat/s32g/include/s32g_ncore.h b/plat/nxp/s32g/include/s32g_ncore.h
similarity index 100%
rename from plat/s32g/include/s32g_ncore.h
rename to plat/nxp/s32g/include/s32g_ncore.h
diff --git a/plat/s32g/include/s32g_pinctrl.h b/plat/nxp/s32g/include/s32g_pinctrl.h
similarity index 100%
rename from plat/s32g/include/s32g_pinctrl.h
rename to plat/nxp/s32g/include/s32g_pinctrl.h
diff --git a/plat/s32g/include/s32g274a_pm.h b/plat/nxp/s32g/include/s32g_pm.h
similarity index 100%
rename from plat/s32g/include/s32g274a_pm.h
rename to plat/nxp/s32g/include/s32g_pm.h
diff --git a/plat/s32g/include/s32g_storage.h b/plat/nxp/s32g/include/s32g_storage.h
similarity index 100%
rename from plat/s32g/include/s32g_storage.h
rename to plat/nxp/s32g/include/s32g_storage.h
diff --git a/plat/s32g/include/s32g_xrdc.h b/plat/nxp/s32g/include/s32g_xrdc.h
similarity index 100%
rename from plat/s32g/include/s32g_xrdc.h
rename to plat/nxp/s32g/include/s32g_xrdc.h
diff --git a/plat/s32g/platform.mk b/plat/nxp/s32g/platform.mk
similarity index 79%
rename from plat/s32g/platform.mk
rename to plat/nxp/s32g/platform.mk
index 1c59aa936..be5e53308 100644
--- a/plat/s32g/platform.mk
+++ b/plat/nxp/s32g/platform.mk
@@ -7,13 +7,13 @@
 include lib/libc/libc.mk
 include lib/libfdt/libfdt.mk
 include lib/xlat_tables_v2/xlat_tables.mk
-include plat/s32g/s32g_common.mk
-include plat/s32g/bl31_sram/bl31_sram.mk
-include plat/s32g/bl31_ssram/bl31_ssram.mk
+include plat/nxp/s32g/s32g_common.mk
+include plat/nxp/s32g/bl31_sram/bl31_sram.mk
+include plat/nxp/s32g/bl31_ssram/bl31_ssram.mk
 
-PLAT_INCLUDES		+= -Iplat/s32g/include \
-			   -Iplat/s32g/bl31_sram/include \
-			   -Iplat/s32g/bl31_ssram/include \
+PLAT_INCLUDES		+= -Iplat/nxp/s32g/include \
+			   -Iplat/nxp/s32g/bl31_sram/include \
+			   -Iplat/nxp/s32g/bl31_ssram/include \
 			   -Iinclude/common/tbbr \
 			   -Iinclude/plat/common \
 			   -Iinclude/plat/arm/common \
@@ -27,41 +27,41 @@ PLAT_INCLUDES		+= -Iplat/s32g/include \
 BL2_AT_EL3		:= 1
 
 
-PLAT_BL_COMMON_SOURCES	+= plat/s32g/s32g_lowlevel_common.S \
-			   plat/s32g/s32g_linflexuart.S \
-			   plat/s32g/s32g_mc_me.c \
-			   plat/s32g/s32g_ncore.c \
-			   plat/s32g/s32g274a_bl_common.c \
-			   plat/s32g/s32g_dt.c \
-			   drivers/nxp/s32g/i2c/s32g274a_i2c.c \
+PLAT_BL_COMMON_SOURCES	+= plat/nxp/s32g/s32g_lowlevel_common.S \
+			   plat/nxp/s32g/s32g_linflexuart.S \
+			   plat/nxp/s32g/s32g_mc_me.c \
+			   plat/nxp/s32g/s32g_ncore.c \
+			   plat/nxp/s32g/s32g_bl_common.c \
+			   plat/nxp/s32g/s32g_dt.c \
+			   drivers/nxp/s32g/i2c/s32g_i2c.c \
 			   drivers/delay_timer/delay_timer.c \
 			   drivers/delay_timer/generic_delay_timer.c \
 			   lib/cpus/aarch64/cortex_a53.S\
 			   ${BL31SRAM_SRC_DUMP} \
 
 
-BL2_SOURCES		+= plat/s32g/s32g_lowlevel_bl2.S \
-			   plat/s32g/s32g274a_bl2_el3.c \
-			   plat/s32g/s32g_clocks.c \
-			   plat/s32g/s32g_pinctrl.c \
-			   plat/s32g/s32g274a_storage.c \
-			   plat/s32g/s32g274a_edma.c \
+BL2_SOURCES		+= plat/nxp/s32g/s32g_lowlevel_bl2.S \
+			   plat/nxp/s32g/s32g_bl2_el3.c \
+			   plat/nxp/s32g/s32g_clocks.c \
+			   plat/nxp/s32g/s32g_pinctrl.c \
+			   plat/nxp/s32g/s32g_storage.c \
+			   plat/nxp/s32g/s32g_edma.c \
 			   drivers/io/io_storage.c \
 			   common/desc_image_load.c \
 			   drivers/mmc/mmc.c \
 			   drivers/nxp/s32g/io/io_mmc.c \
 			   drivers/nxp/s32g/io/io_memmap.c \
 			   drivers/io/io_fip.c \
-			   drivers/nxp/s32g/mmc/s32g274a_mmc.c \
+			   drivers/nxp/s32g/mmc/s32g_mmc.c \
 			   ${DDR_DRV}/ddrss.c \
 			   ${DDR_DRV}/ddrss_firmware.c \
 			   ${DDR_DRV}/ddrss_regconf.c \
 			   ${BL31SSRAM_SRC_DUMP} \
 
-BL31_SOURCES		+= plat/s32g/s32g274a_bl31.c \
-			   plat/s32g/s32g_clocks.c \
-			   plat/s32g/s32g_psci.c \
-			   plat/s32g/s32g274a_pm.c \
+BL31_SOURCES		+= plat/nxp/s32g/s32g_bl31.c \
+			   plat/nxp/s32g/s32g_clocks.c \
+			   plat/nxp/s32g/s32g_psci.c \
+			   plat/nxp/s32g/s32g_pm.c \
 			   plat/common/plat_psci_common.c \
 			   plat/common/plat_gicv3.c \
 			   drivers/arm/gic/v3/gicv3_main.c \
@@ -71,8 +71,8 @@ BL31_SOURCES		+= plat/s32g/s32g274a_bl31.c \
 			   drivers/nxp/s32g/pmic/vr5510.c \
 			   drivers/nxp/s32g/s32g_wkpu.c \
 
-BL31_SOURCES		+= plat/s32g/bl31_lowlevel.S \
-			   plat/s32g/include/plat_macros.S
+BL31_SOURCES		+= plat/nxp/s32g/bl31_lowlevel.S \
+			   plat/nxp/s32g/include/plat_macros.S
 
 BL31_SOURCES		+= ${XLAT_TABLES_LIB_SRCS}
 
diff --git a/plat/s32g/s32g274a_bl2_el3.c b/plat/nxp/s32g/s32g_bl2_el3.c
similarity index 100%
rename from plat/s32g/s32g274a_bl2_el3.c
rename to plat/nxp/s32g/s32g_bl2_el3.c
diff --git a/plat/s32g/s32g274a_bl31.c b/plat/nxp/s32g/s32g_bl31.c
similarity index 99%
rename from plat/s32g/s32g274a_bl31.c
rename to plat/nxp/s32g/s32g_bl31.c
index 5e03f3038..7dbb7a280 100644
--- a/plat/s32g/s32g274a_bl31.c
+++ b/plat/nxp/s32g/s32g_bl31.c
@@ -14,10 +14,10 @@
 #include <psci.h>
 
 #include "drivers/generic_delay_timer.h"
-#include "i2c/s32g274a_i2c.h"
+#include "i2c/s32g_i2c.h"
 #include "platform_def.h"
 #include "pmic/vr5510.h"
-#include "s32g274a_pm.h"
+#include "s32g_pm.h"
 #include "s32g_clocks.h"
 #include "s32g_dt.h"
 #include "s32g_linflexuart.h"
diff --git a/plat/s32g/s32g274a_bl_common.c b/plat/nxp/s32g/s32g_bl_common.c
similarity index 100%
rename from plat/s32g/s32g274a_bl_common.c
rename to plat/nxp/s32g/s32g_bl_common.c
diff --git a/plat/s32g/s32g_clocks.c b/plat/nxp/s32g/s32g_clocks.c
similarity index 100%
rename from plat/s32g/s32g_clocks.c
rename to plat/nxp/s32g/s32g_clocks.c
diff --git a/plat/s32g/s32g_common.mk b/plat/nxp/s32g/s32g_common.mk
similarity index 100%
rename from plat/s32g/s32g_common.mk
rename to plat/nxp/s32g/s32g_common.mk
diff --git a/plat/s32g/s32g_dt.c b/plat/nxp/s32g/s32g_dt.c
similarity index 100%
rename from plat/s32g/s32g_dt.c
rename to plat/nxp/s32g/s32g_dt.c
diff --git a/plat/s32g/s32g274a_edma.c b/plat/nxp/s32g/s32g_edma.c
similarity index 98%
rename from plat/s32g/s32g274a_edma.c
rename to plat/nxp/s32g/s32g_edma.c
index b93bbfca3..3aabaf376 100644
--- a/plat/s32g/s32g274a_edma.c
+++ b/plat/nxp/s32g/s32g_edma.c
@@ -6,7 +6,7 @@
 
 #include <lib/mmio.h>
 #include <assert.h>
-#include "s32g274a_edma.h"
+#include "s32g_edma.h"
 
 /* Determine the optimum data transfer size based on
  * the highest common alignment between 'to', 'from' and 'length'.
diff --git a/plat/s32g/s32g_linflexuart.S b/plat/nxp/s32g/s32g_linflexuart.S
similarity index 100%
rename from plat/s32g/s32g_linflexuart.S
rename to plat/nxp/s32g/s32g_linflexuart.S
diff --git a/plat/s32g/s32g_lowlevel_bl2.S b/plat/nxp/s32g/s32g_lowlevel_bl2.S
similarity index 100%
rename from plat/s32g/s32g_lowlevel_bl2.S
rename to plat/nxp/s32g/s32g_lowlevel_bl2.S
diff --git a/plat/s32g/s32g_lowlevel_common.S b/plat/nxp/s32g/s32g_lowlevel_common.S
similarity index 99%
rename from plat/s32g/s32g_lowlevel_common.S
rename to plat/nxp/s32g/s32g_lowlevel_common.S
index 99045f084..51232d68a 100644
--- a/plat/s32g/s32g_lowlevel_common.S
+++ b/plat/nxp/s32g/s32g_lowlevel_common.S
@@ -7,7 +7,7 @@
 #include <asm_macros.S>
 #include <console_macros.S>
 #include "platform_def.h"
-#include "s32g274a_edma.h"
+#include "s32g_edma.h"
 
 .globl plat_is_my_cpu_primary
 .globl plat_my_core_pos
diff --git a/plat/s32g/s32g_mc_me.c b/plat/nxp/s32g/s32g_mc_me.c
similarity index 100%
rename from plat/s32g/s32g_mc_me.c
rename to plat/nxp/s32g/s32g_mc_me.c
diff --git a/plat/s32g/s32g_ncore.c b/plat/nxp/s32g/s32g_ncore.c
similarity index 100%
rename from plat/s32g/s32g_ncore.c
rename to plat/nxp/s32g/s32g_ncore.c
diff --git a/plat/s32g/s32g_pinctrl.c b/plat/nxp/s32g/s32g_pinctrl.c
similarity index 100%
rename from plat/s32g/s32g_pinctrl.c
rename to plat/nxp/s32g/s32g_pinctrl.c
diff --git a/plat/s32g/s32g274a_pm.c b/plat/nxp/s32g/s32g_pm.c
similarity index 99%
rename from plat/s32g/s32g274a_pm.c
rename to plat/nxp/s32g/s32g_pm.c
index 0680ab632..4ad1c4014 100644
--- a/plat/s32g/s32g274a_pm.c
+++ b/plat/nxp/s32g/s32g_pm.c
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-#include "s32g274a_pm.h"
+#include "s32g_pm.h"
 #include <arch_helpers.h>
 #include <errno.h>
 #include <lib/mmio.h>
diff --git a/plat/s32g/s32g_psci.c b/plat/nxp/s32g/s32g_psci.c
similarity index 100%
rename from plat/s32g/s32g_psci.c
rename to plat/nxp/s32g/s32g_psci.c
diff --git a/plat/s32g/s32g274a_storage.c b/plat/nxp/s32g/s32g_storage.c
similarity index 98%
rename from plat/s32g/s32g274a_storage.c
rename to plat/nxp/s32g/s32g_storage.c
index c39c8d37c..40e56bbd0 100644
--- a/plat/s32g/s32g274a_storage.c
+++ b/plat/nxp/s32g/s32g_storage.c
@@ -9,7 +9,7 @@
 #include <drivers/nxp/s32g/io/io_mmc.h>
 #include <drivers/io/io_memmap.h>
 #include <drivers/io/io_fip.h>
-#include <drivers/nxp/s32g/mmc/s32g274a_mmc.h>
+#include <drivers/nxp/s32g/mmc/s32g_mmc.h>
 #include <assert.h>
 #include <tools_share/firmware_image_package.h>
 #include "s32g_storage.h"
diff --git a/plat/s32g/s32g_xrdc.c b/plat/nxp/s32g/s32g_xrdc.c
similarity index 100%
rename from plat/s32g/s32g_xrdc.c
rename to plat/nxp/s32g/s32g_xrdc.c
-- 
2.17.1

